lib/serializer.js

```diff
@@ -38,3 +38,15 @@ Serializer.prototype.stringToByteArray = function (msg/* : string */) {
 Serializer.prototype.byteArrayToString = function (bytes/* : Uint8Array */) {
   return this.api.GenericMessage.decode(bytes).message
 }
+
+module.exports.serializer = null
+
+module.exports.initSerializer = function (apiFile/* : string */) {
```

Comment:
yeah seems like this should just be a modification to `init` instead of its own method